### PR TITLE
Created the README.md file and downloaded eslint

### PR DESCRIPTION
**Pull request by 95Rawan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,38 @@
...
+* public:
+  - index.html
+  - style.css
+* src:
+  - Components:
+    - Main:
+      - index.js
+    - Result:
+      - index.js
+  * App.js
```
Comment (jema28) [inline README.md:19]: It's good to put the App in the components folder :)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+	"name": "ImageGuessing-Game",
+	"version": "1.0.0",
+	"description": "",
+	"main": "index.js",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1",
```
Comment (jema28) [inline package.json]: Please delete this.

**Review by Muniralsharif — APPROVED** [review]

**Review by jema28 — CHANGES_REQUESTED** [review]

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="logo192.png" />
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
```
Comment (jema28) [inline public/index.html]: You don't need any manifest.json/service worker functions, delete mentions to these from relevant files.

File: src/serviceWorker.js [hunk 4]
```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```
Comment (jema28) [inline src/serviceWorker.js]: Delete this whole file.

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './Components/App';
+import * as serviceWorker from './serviceWorker';
```
Comment (jema28) [inline src/index.js]: Delete this.

File: src/index.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
```
Comment (jema28) [inline src/index.js]: Delete this index.css.

File: src/index.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+import App from './Components/App';
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(<App />, document.getElementById('root'));
+
+// If you want your app to work offline and load faster, you can change
```
Comment (jema28) [inline src/index.js]: Delete service worker, this is used for progressive web apps which we're not doing in this project.

File: src/index.css [hunk 8]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (jema28) [inline src/index.css]: Please delete this file.

**Review by jema28 — CHANGES_REQUESTED** [review]

File: src/Components/App/index.js [hunk 9]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
```
Comment (jema28) [inline src/Components/App/index.js]: Please delete all this starter stuff and replace it with a simple p saying component rendering for now.

**Review by jema28 — CHANGES_REQUESTED** [review]

**Review by jema28 — APPROVED** [review]

**Pull request merged**